xbmc/FileItem.cpp

```diff
@@ -3578,11 +3578,46 @@ std::string CFileItem::GetBaseMoviePath(bool bUseFolderNames) const
 
       URIUtils::GetParentPath(name2, strMovieName);
     }
+
+    // Remove 'Disc' subdirectory so actual title passed to scraper
+    if (CServiceBroker::GetSettingsComponent()->GetSettings()->GetBool(
+            CSettings::SETTING_MYVIDEOS_HANDLEDISCN))
+    {
+      const size_t discPos = GetDiscPath(strMovieName);
+      if (discPos)
+        strMovieName.erase(discPos + 1);
+    }
   }
 
   return strMovieName;
 }
 
+size_t CFileItem::GetDiscPath(const std::string& strPath) const
+{
+  // Handle both Disc and Disk spellings
+  size_t discPos = strPath.rfind("/Disc ");
+  size_t discLen = 6; // Length of disc string (including space and /)
```

Comment:
That's what I meant with a constant. `static constexpr size_t DISC_LEN{6};`

Reply:
Redundant now.